modules/bibdocfile/lib/bibdocfile_webinterface.py

```diff
@@ -164,6 +165,15 @@ def getfile(req, form):
             else:
                 docname = args['docname']
 
+            if CFG_WEBSEARCH_OBELIX_REDIS:
+                try:
+                    obelix = get_obelix()
+                    obelix.log('download_after_search',
+                               user_info,
+                               self.recid)
+                except:
```

Comment:
Bare `except`s are very rarely a good idea. Take a look [here](https://docs.python.org/2/howto/doanddont.html#except), especially the last paragraph.